src/status_im/ui/screens/screens.cljs

```diff
@@ -0,0 +1,771 @@
+
+           ;[Onboarding] Create Password
+           {:name      :create-password
+            :options   {:popGesture         false
+                        :hardwareBackButton {:dismissModalOnPress false
+                                             :popStackOnPress     false}}
+            :insets    {:bottom true}
+            :component onboarding.password/screen}
+
+           ;[Onboarding] Welcome
+           {:name      :welcome
+            :options   {:popGesture         false
+                        :hardwareBackButton {:dismissModalOnPress false
+                                             :popStackOnPress     false}}
+            :insets    {:bottom true}
+            :component onboarding.welcome/welcome}
+
+           ;[Onboarding] Notification
+           {:name      :onboarding-notification
+            :options   {:popGesture         false
+                        :hardwareBackButton {:dismissModalOnPress false
+                                             :popStackOnPress     false}}
+            :insets    {:bottom true}
+            :component onboarding.notifications/notifications-onboarding}
+
+           ;[Onboarding] Recovery
+           {:name      :recover-multiaccount-enter-phrase
+            :insets    {:bottom true}
+            :component onboarding.phrase/enter-phrase}
+           {:name      :recover-multiaccount-success
+            :options   {:popGesture         false
+                        :hardwareBackButton {:dismissModalOnPress false
+                                             :popStackOnPress     false}}
+            :insets    {:bottom true}
+            :component onboarding.phrase/wizard-recovery-success}
+
+           ;;CHAT
+
+           ;Home
+           {:name      :home
+            :component home/home}
+
+           ;Chat
+           {:name          :chat
+            :options       {:topBar {:title        {:component {:name :chat-toolbar :id :chat-toolbar}
+                                                    :alignment :fill}
+                                     :rightButtons (right-button-options :chat :more)}}
+            :right-handler chat/topbar-button
+            :component     chat/chat}
+
+           {:name      :group-chat-profile
+            :insets    {:top false}
+            ;;TODO animated-header
+            :options   {:topBar {:visible false}}
+            :component profile.group-chat/group-chat-profile}
+           {:name      :group-chat-invite
+            ;;TODO parameter in the event
+            :options   {:topBar {:visible false}}
+            :component profile.group-chat/group-chat-invite}
+
+           {:name      :stickers
+            :options {:topBar {:title {:text (i18n/label :t/sticker-market)}}}
+            :component stickers/packs}
+
+           {:name      :stickers-pack
+            :component stickers/pack}
+
+           {:name      :notifications-center
+            ;;TODO custom nav
```

Comment:
There are multiple TODOs in this NS. Do you intend to address these before merge?

Reply:
no, for later